src/lib/components/Toggle/root.svelte

```diff
@@ -1,24 +1,29 @@
 <script lang="ts" context="module">
-	import { controllableState } from '$lib/helpers/controllableState';
 	import type { BaseProps } from '$lib/types';
 
-	export type ToggleProps = BaseProps & {
+	export type ToggleProps = BaseProps<HTMLButtonElement> & {
+		/** The controlled pressed state of the toggle. */
 		pressed?: boolean;
+		/** When true, prevents the user from interacting with the toggle. */
 		disabled?: boolean;
+		/** The pressed state of the toggle when it is initially rendered.
+		 *  Use when you do not need to control its pressed state. */
+		defaultPressed?: boolean;
 	};
 </script>
 
 <script lang="ts">
 	type $$Props = ToggleProps;
 
-	export let pressed: $$Props['pressed'] = false;
+	export let defaultPressed: $$Props['defaultPressed'] = undefined;
+	export let pressed: $$Props['pressed'] = defaultPressed ?? false;
 	export let disabled: $$Props['disabled'] = false;
```

Comment:
This will create a different behaviour than on Collapsible. On Collapsible, if we change the defaultOpen prop *after* render, it changes open. Here, it'll only change it on mount. Maybe it shouldn't tbh! Could you change the Collapsible line for me?
```ts
// Before
$: $writableOpen = open || defaultOpen;
// After
$: $writableOpen = open;
```

Reply:
There's also another issue. If the user sets the pressed value, setting defaultPressed here won't do a thing. Maybe we should set pressed to defaultPressed onMount (if defaultPressed isn't undefined)

Reply:
Hmm good question (and worth figuring out the right behavior for consistently)

The Radix docs are `The pressed state of the toggle when it is initially rendered. Use when you do not need to control its pressed state.`.

I was wondering when making the change if `defaultPressed` (and alike in other components) even is that meaningful in svelte? If I werent trying to match the react api, i'd just default it on the `pressed` property instead. We could keep it for api consistency, but exactly the situation of setting both `pressed` and `defaultPressed` makes me think its actually more confusing to have them.

Reply:
Yeah, tbh in React, it makes sense to have both. Because if you pass in `pressed={false}`, it'll always be false. But in Svelte, that's not the case! So maybe we could do away, and remove `defaultPressed` everywhere. Do you agree?

Reply:
Yeah, I'm leaning that way. No reason to inherit react complexities for the sake of complexities.